src/dom/geometry/domRectRe.re

```diff
@@ -1,15 +1,19 @@
-external top : Dom.domRect => int = "" [@@bs.get];
+type t = Dom.domRect;
 
-external bottom : Dom.domRect => int = "" [@@bs.get];
+external make : int => int => int => int => t = "DOMRect" [@@bs.new]; /* experimental */
```

Comment:
Could you also add labels to these arguments? With four arguments of the same type it's very easy to mistake them.